libraries/protocol/include/golos/protocol/exceptions.hpp

```diff
@@ -334,6 +335,12 @@ namespace golos {
             cannot_specify_owner_key_unless_creating_account,
             witness_must_be_created_before_minning,
 
+            // worker proposals
+            too_early_creating_worker_proposal,
```

Comment:
the idea was to disable too late proposals. and current implementation have no that check at all, so this can be removed. same about `GOLOS_WORKER_PROPOSAL_AFTER_POST_SEC`